docs/using.md

```diff
@@ -6,6 +6,7 @@
 - [How do I run a Pipeline?](#running-a-pipeline)
 - [How do I run a Task on its own?](#running-a-task)
 - [How do I troubleshoot a PipelineRun?](#troubleshooting)
+- [How do I follow logs](../test/logs/README.md)
```

Comment:
I guess once we have #107 implemented this will change a bit :thinking:

Reply:
+ @tejal29

Reply:
This PR is for tracking logs from pod directly. #107 issue is about uploading logs to external bucket. I do not think they are related. After #107 I could see another "How do I access logs of taskruns?" point which should hold instructions on how to fetch logs from bucket.